app/QueryManager.py

```diff
@@ -1,5 +1,6 @@
 from pyspark.sql import Window
-from pyspark.sql.functions import col, date_format, desc, max
+from  pyspark.sql.functions import col, date_format, desc, max, sum, format_number, avg, lit, asc
```

Comment:
please use import of fucntions as alias at least for fucntions like max, sum etc because it gets confusing when these functions overwrite default ones